pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>code-coverage-api</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0</version>
```

Comment:
I doubt it's what you want. Maven release plugin does it automatically

Reply:
Yeah, you are right, I forget that.